parser.py

```diff
@@ -21,38 +23,32 @@ def get_initials(name):
 for rownum in range(0, sh.nrows):
  person = OrderedDict()
  row_values = sh.row_values(rownum)
- 
- 
+
+
  for values in row_values:
   if values=='':
    i=row_values.index(values)
    row_values[i]="null"
- 
- #if 
+
+ #if
  # person['head']= TRUE
  person['name'] = row_values[0]
  person['initials'] = get_initials(person['name'])
  person['party'] = row_values[1]
  person['occupation'] = row_values[2]
- person['twitter'] = row_values[3]	
- person['facebook'] = row_values[4]	
+ person['twitter'] = row_values[3]
+ person['facebook'] = row_values[4]
  person['section'] = row_values[5]
  person['head'] = False
- 
+
  if temp!=row_values[5]:
   person['head'] = True
- 
+
  temp = row_values[5]
 
  people_list.append(person)
 
 #print people_list
 
-j = json.dumps(people_list).encode('utf8')
-
-with codecs.open("data.json", "w", encoding='utf-8') as f:
- f.write(j)
-
-
-
-
+with codecs.open("data.json", "w", "utf-8") as f:
+    json.dump(people_list, f, ensure_ascii=False)
```

Comment:
το `ensure_ascii` είναι το κόλπο. από το http://stackoverflow.com/questions/18337407/saving-utf-8-texts-in-json-dumps-as-utf8-not-as-u-escape-sequence